include/swift/AST/FineGrainedDependencyFormat.h

```diff
@@ -0,0 +1,106 @@
+//===---- FineGrainedDependencyFormat.h - swiftdeps format ---*- C++ -*-======//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SWIFT_AST_FINEGRAINEDDEPENDENCYFORMAT_H
+#define SWIFT_AST_FINEGRAINEDDEPENDENCYFORMAT_H
+
+#include "llvm/Bitcode/RecordLayout.h"
+#include "llvm/Bitstream/BitCodes.h"
+
+namespace llvm {
+class MemoryBuffer;
+}
+
+namespace swift {
+
+class DiagnosticEngine;
+
+namespace fine_grained_dependencies {
+
+class SourceFileDepGraph;
+
+using llvm::BCFixed;
+using llvm::BCVBR;
+using llvm::BCBlob;
+using llvm::BCRecordLayout;
+
+const unsigned char FINE_GRAINED_DEPDENENCY_FORMAT_SIGNATURE[] = {'D', 'E', 'P', 'S'};
+
+const unsigned FINE_GRAINED_DEPENDENCY_FORMAT_VERSION_MAJOR = 1;
```

Comment:
No, it's the version number of the new binary format, not the version number of the overall dependency format. (In which case, it would be 3, with the fine-grained format v2 :-) )

Reply:
OK, I guess I was misled by the name "fine grained dependency format version", which implied to me that the number should be 2, since this is the second one such. But I can't think of anything that would break by leaving it as 1, since  an old driver should reject this new format.